dashboard/app.py

```diff
@@ -100,7 +100,8 @@ def favicon():
 
 @app.route("/")
 def home():
-    return redirect("/dashboard", code=302)
+    redirect_url = "https://sso.mozilla.com/dashboard"
```

Comment:
This won't work because we host at multiple URLs.  Must inherit the base url from the CONFIG object in this string.

Reply:
Ah, so steal from app.config.get("SERVER_NAME"), thanks!

Reply:
This setting should also only be applied in the production env.

Reply:
I'll mimic the logic here (but invert the conditional)

https://github.com/mozilla-iam/sso-dashboard/blob/master/dashboard/app.py#L263-L267

Reply:
Fix should now only affect sso.mozilla.com